board/safety/safety_subaru.h

```diff
@@ -68,6 +68,8 @@ AddrCheckStruct subaru_gen2_addr_checks[] = {
 #define SUBARU_GEN2_ADDR_CHECK_LEN (sizeof(subaru_gen2_addr_checks) / sizeof(subaru_gen2_addr_checks[0]))
 addr_checks subaru_gen2_rx_checks = {subaru_gen2_addr_checks, SUBARU_GEN2_ADDR_CHECK_LEN};
 
+#define STEER_ANGLE_MEAS_FACTOR -0.0217
```

Comment:
can you put in `SteeringLimits.angle_deg_to_can`?

Reply:
used SUBARU_ prefix. This PR doesn't have SteeringLimits, that will come with the forester PR.